### ISSUE TITLE
 'NoneType' object has no attribute 'transaction' When adding dataset 

### ISSUE THREAD

**Issue by hussnainwithss** [ckan/ckanext-xloader#147]
Setup: 
Fresh CKAN 2.9 setup with xloader on ubuntu 20.04 Locally

Steps to reproduce:
1. Go to add dataset screen
2. fill relevant meta info 
3. select upload file and upload following csv
4. 500 internal server error reported 

Expected Behavior:
The file should be uploaded and the xloader job should start pushing to the datastore.

![image](https://user-images.githubusercontent.com/47157583/142431967-831a9f6a-9489-4783-bad3-d6889d3ca68c.png)
![image](https://user-images.githubusercontent.com/47157583/142432129-9a08dbd8-d6e5-4e05-abb2-d24bee5dc082.png)

**Comment by tino097** [comment]
Hello @hussnainwithss 
can you please check your ckanext-xloader version as this issue was [fixed ](https://github.com/ckan/ckanext-xloader/pull/140). If you are still experiencing the issue, please repon the issue in the xloader repository. Thanks

**Comment by daschoe** [comment]
I had the same problem: I installed xloader with pip first and then noticed that that gives one version 0.8.0 where this issue wasn't fixed. But installing over git clone fixes that of course...

**Comment by categulario** [comment]
I don't think this issue should be closed until a new version (0.9) is uploaded to pypi because the readme calls for an installation from there, thus causing the error here described.
